doc/requirements.txt

```diff
@@ -32,6 +32,9 @@ torchvision==0.12.0; sys_platform == "darwin" and python_version == "3.10"
 jinja2==3.0.3
 retworkx==0.11.0
 networkx==2.6
+dill~=0.3.5.1
+zstd~=1.5.2.6
+requests~=2.28.1
```

Comment:
Where this is needed?

Reply:
This will be required for the next PR, the list and load functions.